ia/src/message.py

```
@@ -0,0 +1,50 @@
+    try:
+        message_crypted: List[Tuple[bytes, bytes]] = []
+        if len(message) > 0:
+            for index in range(len(message)):
+                if message[index] != "\n":
+                    message_crypted_temp = encrypt(message[index])
+                    message_crypted.append(message_crypted_temp)
+                else:
+                    e = "Empty message"
+        e = "No messages on list"
+        return message_crypted
+    except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)